cves/CVE-2015-6767.yml

```
@@ -4,21 +4,21 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE: 
+CWE: CWE-416
```

The CWE value should just be the CWE ID number.
```suggestion
CWE: 416
```